hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/s3/security/S3GetSecretRequest.java

```diff
@@ -168,9 +168,12 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
                 assignS3SecretValue = null;
               }
             } else {
-              // Found in S3SecretTable.
+              // Found in S3SecretTable. No secret returned.
               awsSecret.set(s3SecretValue.getAwsSecret());
-              assignS3SecretValue = null;
+              return new S3GetSecretResponse(null,
+                      ozoneManager.getS3SecretManager(),
+                      createErrorOMResponse(omResponse, new
+                              OMException(OMException.ResultCodes.INVALID_REQUEST)));
```

Comment:
Please add a new ResultCodes，say “S3_SECRET_ALREADY_EXIST”.

Reply:
@Tejaskriya , thanks for working on this.  Would you add a robot test for s3 secret under hadoop-ozone/dist/src/main/smoketest/security?

Reply:
Yes, I have made the changes and added a robot test too